pkg/scheduler/plugins/nodeReservation/plugin.go

```diff
@@ -0,0 +1,207 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package nodeReservation
```

Comment:
Package and directory names should be all lowercase. Avoid introducing some unnecessary problems on some case-insensitive OS.

Reply:
got it